### ISSUE TITLE
Allow users to dismiss annotation adder without moving pointer or keyboard focus (WCAG 2.1 criterion 1.4.13)

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#1128]
[1.4.13 Content on Hover or Focus](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus) stipulates that content that appears on hover or focus must be dismissable without moving pointer hover or keyboard focus, unless the additional content communicates an input error or does not obscure or replace other content

We received a "partially supports" for this because the annotation adder can't be dismissed without moving the pointer or keyboard focus, and the presence of the adder can (but does not always) obscure other content on the page:

![image](https://user-images.githubusercontent.com/12087705/89078111-b9b9e180-d351-11ea-9079-f4f81c4576a1.png)

The IDRC suggested allowing users to dismiss the adder using the Esc key, as that's the most common mechanism for dismissing popups with just a keyboard.
